src/relayFeeCalculator/chain-queries/zksync.ts

```diff
@@ -26,3 +26,27 @@ export class ZkSyncQueries extends QueryBase {
     );
   }
 }
+
+export class zkSyncGoerliQueries extends QueryBase {
+  constructor(
+    provider: providers.Provider,
+    symbolMapping = TOKEN_SYMBOLS_MAP,
+    spokePoolAddress = "0x863859ef502F0Ee9676626ED5B418037252eFeb2",
```

Comment:
Can we load any of these variables from existing files? Like TOKEN_SYMBOLS_MAP?

Or at least comment where we find these from so in the future we can more easily modify

Reply:
This is a good call - I'd been meaning to do that later, but it's a really good idea to get it done now. The first part is here: https://github.com/across-protocol/sdk-v2/pull/367/commits/69e8ea01971ec504d9f1ebce3f31206f7ac78209

I'll follow up with a commit to make the SpokePool address resolution dynamic as well.